Add robot playwright library (#3216)
SFDO-Tooling/CumulusCI: docs/robot_playwright.rst

Starting with CumulusCI version 3.59.0, we are providing experimental
support for Playwright and the Browser library in CumulusCI.

> In CumulusCI version 3.60 we've reorganized our keywords so that

Reviewer: ```suggestion
In CumulusCI 3.60, we've reorganized our keywords so that
```

a test can import the API and performance keywords without importing
Selenium keywords. To use Playwright-based keywords, import the resource file
`SalesforcePlaywright.robot

> <Keywords.html#file-cumulusci/robotframework/SalesforcePlaywright.robot>`_

Reviewer: ```suggestion
<Keywords.html#file-cumulusci/robotframework/SalesforcePlaywright.robot>`_,
```

> which imports the non-selenium keywords along with the keywords in

Reviewer: ```suggestion
which imports the non-Selenium keywords along with the keywords in
```

the Browser library
<https://marketsquare.github.io/robotframework-browser/Browser.html>`_

> To initialize playwright support in a test suite, import the

Reviewer: ```suggestion
To initialize Playwright support in a test suite, import the
```

> SalesforcePlaywright.robot resource file as shown in the following

Reviewer: ```suggestion
``SalesforcePlaywright.robot`` resource file as shown in the following
```

> example. It will import the Browser library, and defines the keywords

Reviewer: ```suggestion
example. It imports the Browser library and defines the keywords
```
